Cli-CredentialHelper/NativeMethods.cs

```diff
@@ -332,6 +332,46 @@ public enum FileShare : uint
             Delete = 0x00000004
         }
 
+        public enum FileType
+        {
+            /// <summary>
+            /// Either the type of the specified file is unknown, or the function failed.
+            /// </summary>
+            Unknown = 0x0000,
+            /// <summary>
+            /// The specified file is a disk file.
+            /// </summary>
+            Disk = 0x0001,
+            /// <summary>
+            /// The specified file is a character file, typically an LPT device or a console.
+            /// </summary>
+            Char = 0x0002,
+            /// <summary>
+            /// The specified file is a socket, a named pipe, or an anonymous pipe.
+            /// </summary>
+            Pipe = 0x0003,
+            /// <summary>
+            /// Unused.
+            /// </summary>
+            Remote = 0x8000,
+        };
+
+        public enum StandardHandleType
+        {
+            /// <summary>
+            /// The standard input device. Initially, this is the console input buffer, CONIN$.
+            /// </summary>
+            Input = -10,
+            /// <summary>
+            /// The standard output device. Initially, this is the active console screen buffer, CONOUT$.
+            /// </summary>
+            Output = -11,
+            /// <summary>
+            /// The standard error device. Initially, this is the active console screen buffer, CONOUT$.
```

Comment:
Per [MSDN GetStandardHandle](https://msdn.microsoft.com/en-us/library/windows/desktop/ms683231(v=vs.85).aspx) it's `CONOUT$` :raised_hands: